src/nbl/video/IQueue.cpp

```diff
@@ -3,6 +3,10 @@
 #include "nbl/video/ILogicalDevice.h"
 #include "nbl/video/TimelineEventHandlers.h"
 
+#define LOG_FUNCTION logger->log
+#define LOG(SEVERITY, FORMAT, ...) LOG_FUNCTION("FORMAT [%s - %s:%d]", SEVERITY, __VA_ARGS__, __FUNCTION__, __FILE__, __LINE__);
+#define LOG_ERROR(FORMAT, ...) LOG(nbl::system::ILogger::ELL_ERROR, FORMAT, __VA_ARGS__)
```

Comment:
thought about moving the macros into two headers (X-macros) one to define them and another to undef?